src/rt/sched/cown.h

```diff
@@ -1199,10 +1204,16 @@ namespace verona::rt
 
         // Reschedule the other cowns.
         for (size_t s = 0; s < (senders_count - 1); s++)
-          senders[s]->schedule();
+          if (senders[s])
```

Comment:
Some braces might not hurt here, or at least would have stopped me from tripping up.  Yes, `for() if() s;` generates the right AST, but `for() { if() s; }` is a bit clearer to read.